src/styles/blocks/nav-list.scss

```diff
@@ -0,0 +1,32 @@
+@import "../utils/variables";
+
+.nav-list {
+  list-style-type: none;
+  display: flex;
+  margin: 0;
```

Comment:
reset default values in the `reset `file

Reply:
https://github.com/mate-academy/layout_catalog/pull/2594/files#r1119807642
not fixed